test/cmd/authentication.sh

```diff
@@ -9,11 +9,20 @@ source "${OS_ROOT}/hack/lib/init.sh"
 os::log::stacktrace::install
 trap os::test::junit::reconcile_output EXIT
 
+
+os::test::junit::declare_suite_start "cmd/authentication/login"
+echo "login warnings: ok"
+os::test::junit::declare_suite_end
+
 os::test::junit::declare_suite_start "cmd/authentication"
 
 os::test::junit::declare_suite_start "cmd/authentication/scopedtokens"
 os::cmd::expect_success 'oadm policy add-role-to-user admin scoped-user'
 
+# make sure we handle invalid config file destination
+os::cmd::expect_failure_and_text "oc login '${KUBERNETES_MASTER}' -u test -p test --config=/src --insecure-skip-tls-verify" 'KUBECONFIG is set to a file that cannot be created or modified'
```

Comment:
I believe this belongs to the suite you started in line 13 (which is doing nothing).